src/Claim.test.ts

```diff
@@ -41,31 +54,117 @@ const PrivateKeyEAP = 'KxuZJmgVAipi9hfYXHTyGYmmhkbG7fBzmkyVnj6t9j9rDR1nN1vN'
 
 describe('Claim', async (should: any) => {
   const { assert } = should('')
+  const returnError = (err: Error) => err
 
   {
-    const claim = createClaim(Key.privateKey, ClaimType.Work, TheRaven.attributes)
+    const claim = makeClaim({
+      name: TheRaven.attributes.name,
+      author: TheRaven.attributes.author,
+    })
+
+    const canonicalDoc = await expandAndCanonizeClaim(claim).catch(returnError)
 
     assert({
-      given: 'the public key of a Claim',
-      should: 'be equal to public key of key',
-      actual: claim.publicKey,
-      expected: Key.publicKey,
+      given: 'a claim',
+      should: 'create a canonical string document',
+      actual: canonicalDoc,
+      expected: expectedCanonicalDoc,
     })
   }
 
   {
-    const claim = createClaim(Key.privateKey, ClaimType.Work, TheRaven.attributes)
+    const work1: Claim = makeClaim({
+      name: TheRaven.attributes.name,
+      author: TheRaven.attributes.author,
+    })
+
+    const work2: Claim = makeClaim({
+      author: TheRaven.attributes.author,
+      name: TheRaven.attributes.name,
+    })
+
+    const canonicalDocument1 = await expandAndCanonizeClaim(work1).catch(returnError)
+    const canonicalDocument2 = await expandAndCanonizeClaim(work2).catch(returnError)
 
     assert({
-      given: 'the result of isValidSignature of Claim with a valid signature',
-      should: 'should return true',
-      actual: isValidSignature(claim),
+      given: 'two claims with disordered keys',
+      should: 'have the same canonical document',
+      actual: canonicalDocument1 === canonicalDocument2,
       expected: true,
     })
   }
 
   {
-    const claimId = getClaimId(TheRaven)
+    const work1: Claim = makeClaim({
+      name: TheRaven.attributes.name,
+      author: TheRaven.attributes.author,
+    })
+
+    const work2: Claim = makeClaim({
+      Author: TheRaven.attributes.author,
+      NAME: TheRaven.attributes.name,
+    })
+
+    const canonicalDocument1 = await expandAndCanonizeClaim(work1).catch(returnError)
+    const canonicalDocument2 = await expandAndCanonizeClaim(work2).catch(returnError)
+
+    assert({
+      given: 'two claims with keys casing',
+      should: 'have the same canonical document',
+      actual: canonicalDocument1 === canonicalDocument2,
+      expected: true,
+    })
+  }
+
+  {
+    const claim = makeClaim({
+      name: TheRaven.attributes.name,
+      author: TheRaven.attributes.author,
+    })
+
+    const id = await getClaimId(claim)
+
+    assert({
+      given: 'A claim',
+      should: 'generate an id for the claim',
+      actual: id,
+      expected: '31f4dd63047c5a053aead0dbb9dd24a08b2b97c153f7f80e03d839371f558b32',
+    })
+  }
+
+  {
+    try {
```

Comment:
Why are we wrapping `try/catch` around the entire assert block? Does the `createClaim(Key.privateKey, ClaimType.Work, TheRaven.attributes).catch(err => err)` trick not work here?